### ISSUE TITLE
Support wildcards

### ISSUE THREAD

**Issue by notramo** [nroi/waitforfile#3]
Do you plan to add support for wildcards?
```bash
waitforfile "directory/*.txt"
```

**Comment by nroi** [comment]
I can imagine that something like this would be useful, but supporting wildcard matching with this:

    waitforfile "directory/*.txt"

would be ambiguous. What if a user actually wants to wait for a file named `*.txt`, rather than using a wildcard?

I would probably include a command line switch like `--pattern` which can be used to match against a POSIX regex.

**Comment by notramo** [comment]
@nroi Yes, you are right. Pattern matching should be enabled explicitly with a switch.
